core/src/test/java/io/questdb/test/griffin/engine/AbstractFunctionFactoryTest.java

```diff
@@ -417,6 +418,10 @@ private void printConstant(int type, StringSink sink, Object value) {
         }
     }
 
+    protected static Utf8Sequence utf8(CharSequence value) {
```

Comment:
nit: worth moving this method to `AbstractCairoTest`, so that we can use in all tests.